### DISCUSSION TITLE
Hikvison IPC-D140H: feed lagg behind

### DISCUSSION THREAD

**Discussion by arkof** [Sunoo/homebridge-camera-ffmpeg#1176] [Q&A] [answered]
Hi
Im relatively new in homebridge.

I'v installed the camera-ffmpeg plugin and added my 2 Hikvision IPC-D140H cameras like the configuration below.
The feed works fine if I view the rtsp feed via VLC. But when viewing it via HomeKit; every second of the camera feed is displayed 3-5 seconds in the app. So the feed lagg behind pretty soon. I can see this on the clock in the camera feed.

I have tryed play around with FPS, bitrate, codec and so on without success.
I don't get any errors in the logg in Homebridge.

Can anyone point me in the right direction on what I can troubleshoot more?

HomeBridge running on RPI config:

`"name": "Camera FFmpeg", "cameras": [ { "name": "Uppfart", "unbridge": false, "videoConfig": { "source": "-rtsp_transport tcp -re -i rtsp://XXXX:XXXXX@192.168.1.210/Streaming/Channels/101", "maxWidth": 800, "maxHeight": 600, "maxFPS": 25, } }, { "name": "Baksidan", "videoConfig": { "source": "-rtsp_transport tcp -re -i rtsp://XXXX:XXXXX@192.168.1.211/Streaming/Channels/101" } } ], "platform": "Camera-ffmpeg" } ]`

**Comment by arkof** [comment, marked as answer]
Found a simple fix.
Adding this to the cameras config:
`"vcodec": "copy"`

**Reply by Jasonrobot00** [reply]
Hey sorry im new to this and i have no idea where to put that in. could you please elaborate.

im having an issue with my cameras, they go fuzzy and the screen turns grey occasionally, ive lowered the packet size, and that seems to help a little bit. i also think it might be the raspberry pi im running it on (its a 3) - wondering if that might be what it is because the raspberry pi gets to 80 degrees pretty quickly. 

Any help would be appreciated
